packages/flutter/lib/src/material/material.dart

```diff
@@ -76,6 +85,15 @@ abstract class MaterialInkController {
 
 /// A piece of material.
 ///
+/// The Material widget is responsible for:
+/// 1. Clipping - Material clips its widget sub-tree to the shape specified by
+///               [type] and [borderRadius].
+/// 2. Elevation - Material elevates its widget sub-tree on the Z axis by
+///                [elevation] pixels, and draws the appropriate shadow.
+/// 3. Ink effects - Material shows ink effects implemented by [InkFeature]s
+///                  like [InkSplash] and [InkHighlight] below its children.
```

Comment:
(also, consider using colons instead hyphens for consistency with other docs)

Reply:
done